understanding/20/use-of-color.html

```diff
@@ -63,6 +63,17 @@ <h2>Intent of Use of Color</h2>
 
       </div>
 
+     <div class="note">
+       
+       <p>This criterion does not apply to situations where color has <em>not</em> been used to convey information, indicate an action,
+         prompt a response, or distinguish a visual element. For instance, a hyperlink which has been styled to appear no
+         different than the static text surrounding it. As there would be no color differentiation between the actionable 
```

Comment:
The construct 

> As there would be no color differentiation between the actionable hyperlink text and the static text adjacent to it, this would be a poor experience for all sighted users, but not a failure of this Success Criterion.

seems a bit difficult to parse. I suggest a simpler version:

> No perceivable difference between the link and the text around creates a poor experience for all sighted users, but is not a failure of this Success Criterion.

Reply:
@detlevhfischer I'd add "it" to read "...text around it creates..."

Reply:
Actually, I'd suggest rewording a bit more. How about...
> For example, if link text has been styled to appear no different than the text around it. As there would be no color differentiation between the actionable link and static text, this would be a a poor experience for all sighted users, but not a failure of this Success Criterion.